openvox/vpat.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: page
+title: Voluntary Product Accessibility Template
+subsection: openvox
+---
+
+The Voluntary Product Accessibility Template (VPAT) is a document that describes how an information technology (IT) product, such as software, hardware, or web-based applications, meets the accessibility standards defined by Section 508 of the Rehabilitation Act. This VPAT describes the accessibility features of the OpenVox open-source project.
```

Comment:
I'd emphasize it's the US Rehabilitation Act. I'd assume most non-US readers are unfamiliar with it.

Please also insert a newline between sentences.
```suggestion
The Voluntary Product Accessibility Template (VPAT) is a document that describes how an information technology (IT) product, such as software, hardware, or web-based applications, meets the accessibility standards defined by Section 508 of the Rehabilitation Act.
This VPAT describes the accessibility features of the OpenVox open-source project.
```